packages/salesforcedx-vscode-core/package.nls.json

```diff
@@ -5,6 +5,7 @@
   "force_auth_access_token_authorize_org_text": "SFDX: Authorize an Org using Session ID",
   "force_auth_logout_all_text": "SFDX: Log Out from All Authorized Orgs",
   "force_auth_logout_default_text": "SFDX: Log Out from Default Org",
+  "force_open_documentation_text": "SFDX: Open Documentation",
```

Comment:
I spoke with @sbudhirajadoc and she would like the command to be, "SFDX: Open Documentation"